roles/nginxplus/files/conf/http/abid-staging.conf

```diff
@@ -0,0 +1,46 @@
+# Ansible managed
+proxy_cache_path /data/nginx/abid-staging/NGINX_cache/ keys_zone=abid-stagingcache:10m;
+
+upstream abid-staging {
+    zone abid-staging 64k;
+    server abid-staging1.princeton.edu resolve;
+    server abid-staging2.princeton.edu resolve;
```

Comment:
Is there an abid-staging2? Do we need it?

Reply:
nice catch! updated.